lib/Raven/Client.php

```diff
@@ -1602,7 +1620,11 @@ public function close_curl_resource()
 
     public function setAllObjectSerialize($value)
     {
-        $this->serializer->setAllObjectSerialize($value);
-        $this->reprSerializer->setAllObjectSerialize($value);
+        if (method_exists($this->serializer, 'setAllObjectSerialize')) {
```

Comment:
Accordind to the comment left below about the `serialize` function definition, we could use the `$context` parameter to pass additional options to the serializer (in this case about serializing all objects) so that we don't need to rely on some methods not even defined in the interface